server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/DLPConfigurationRoutesTest.java

```diff
@@ -407,7 +408,9 @@ void putShouldUpdateTheConfigurations() {
             .extract()
                 .body().asString();
 
-            assertThatJson(retrievedBody).isEqualTo(updatedBody);
+            assertThatJson(retrievedBody)
+                .when(Option.IGNORING_ARRAY_ORDER)
```

Comment:
This one won't be easy to handle:

 - Unordered collection is used to store the state (set)
 - Furthermore, order is not handled by the events 

Either we:
 - Don't care. After all, rules are indempotent since we have two actions (ignore or quarantine). The only diference is "which rule matches".
 - Create a ticket, and do it later
 - Completly clear the state after each update. Seems like something you tried to avoid.
 - Or implement crazy logic to handle order